lib/v2/generator.js

```diff
@@ -1,6 +1,24 @@
 const AbstractApiGenerator = require('../openapi');
 const utils = require('../utils');
 
+function idPathParameters (idName, idType, description) {
+  const idNames = Array.isArray(idName) ? idName : [idName];
+  const idTypes = Array.isArray(idType) ? idType : [idType];
+  const params = [];
+
+  for (const [key, name] of Object.entries(idNames)) {
```

Comment:
Why you changed this from array to object iteration?

Reply:
It's array iteration with key & value. it works with NodeJS 8 and up, so the iteration in the v1 branch is using a normal `for` loop. if you prefer you can apply that instead in the master branch.

Reply:
Even though arrays are also objects, you should never call a function for objects on it, that transforms it into an array :)

Ok so I will merge the prepared branch.
Thank you very much for the PR.